src/components/UI/SourceSelect/SourceSelect.theme.js

```diff
@@ -59,14 +59,15 @@ export const SourceSelectTheme = ({children}) => {
       MuiList: {
         styleOverrides: {
           root: {
-            width: '240px',
-            padding: '8px 16px'
+            width: '240px !important',
+            padding: '8px 16px !important'
```

Comment:
here too, and I saw more `!important`s so maybe in more places it's redundant

Reply:
these are needed for small screen where MUI changes the width and padding in the JS